go-controller/pkg/ovn/ovn.go

```diff
@@ -375,6 +375,7 @@ func (oc *Controller) WatchPods() error {
 		DeleteFunc: func(obj interface{}) {
 			pod := obj.(*kapi.Pod)
 			oc.deleteLogicalPort(pod)
+			deletePodConntrack(pod.Status.PodIP)
```

Comment:
I haven't dug into the code yet, but this looks a little odd.  `deleteLogicalPort()` is somethign I'd only expect to run on a single host, but deleting conntrack state would be something that needs to happen across the whole cluster.  Where exactly does this run?

Reply:
@russellb you are right this will only run on the master. I moved conntrack deletion to PlatformSpecificCleanup() in the cni code and it should now run on all nodes